modules/nf-core/cooltools/eigscis/main.nf

```diff
@@ -0,0 +1,46 @@
+process COOLTOOLS_EIGSCIS {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
Does this module use more than one core?

Reply:
I will label this with 'process_single'

Reply:
Allright be aware that the memory also lowers with the `process_single` label